app/javascript/packages/address-search/components/no-in-person-locations-display.tsx

```diff
@@ -1,14 +1,18 @@
 import { getAssetPath } from '@18f/identity-assets';
 import { t } from '@18f/identity-i18n';
 
-function NoInPersonLocationsDisplay({ address }) {
+interface NoInPersonLocationsDisplayProps {
+  address: string;
+}
+
+function NoInPersonLocationsDisplay({ address }: NoInPersonLocationsDisplayProps) {
   return (
     <div className="grid-row grid-gap grid-gap-1">
       <img
         className="grid-col-2 margin-top-3"
         alt={t('image_description.info_pin_map')}
         width={65}
-        height={64.6}
+        height={65}
```

Comment:
Request to change the value from 64.6 (value in mocks) to 65 was made [here](https://github.com/18F/identity-idp/pull/9145#discussion_r1319001204).